### DISCUSSION TITLE
config/plugins/markdown-power/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#322] [docs-comment]
# config/plugins/markdown-power/

概述 为 主题 提供 Markdown 增强功能。 关联插件: @vuepress-plume/plugin-md-power 配置 默认配置： 功能 嵌入 PDF 插件默认不启用该功能，你需要手动设置 pdf 为 true 语法: 请查看 iconify 图标 插件默认不启用该功能，你需要手动设置 icons 为 true 得益于 iconify, ...

https://theme-plume.vuejs.press/config/plugins/markdown-power/

**Comment by dingyuqi** [comment]
conify 图标 章节下的超链接失效

**Reply by dingyuqi** [reply]
完整使用文档

**Reply by pengzhanbo** [reply]
已修复，感谢指出。

**Reply by dingyuqi** [reply]
许多 “完整使用文档” 的链接已经失效。

**Reply by pengzhanbo** [reply]
已修复，感谢指出
